week-1/Homework/extra/2-piping.js

```diff
@@ -16,26 +16,28 @@
   the final result to the variable goodCode
 */
 
-function add() {
+function add(a,b) {
+  return Math.round((a + b)*100)/100;
 
+  //IT WAS A BIT MORE DIFFICULT THEN I THOUGHT!:)
 }
 
```

Comment:
Above function implementation  is very clever and it works. 

But can you think of few reasons why someone might say this code is bad, even though it works and passes the test.

Reply:
spaghetti code, need more explanation